src/main/java/frc/robot/subsystems/WristSubsystem.java

```diff
@@ -72,10 +71,10 @@ public void encWrisDutyCycleEncoderaise() {
     // double degrees = encWrisDutyCycleEncoderot.get()*360;
     // System.out.println(degrees);
 
-    double rotation = encoderaise.getPosition();
+    double rotation = encoderaise.getPosition() / 40;
```

Comment:
Comment here and below on why 40 or 0.5 is being used or how they are calculated.